### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by mpsijm**
Changes:

- [README.md: Add steps for development](https://github.com/WISVCH/nwerc-2022/pull/2/commits/2609b3731c52849c18ac11bed7c6822155a59c56)
- [Copyright: change from default "NWERC 2022" to "WISV CH"](https://github.com/WISVCH/nwerc-2022/pull/2/commits/01458af2994477debc6a975011c91e8079499f24)
- [Fix some small typos](https://github.com/WISVCH/nwerc-2022/pull/2/commits/8ff982e840b449b3f420adbeda246df5c6fb1164)
- [contest.md: Fix broken URLs to previous contests](https://github.com/WISVCH/nwerc-2022/pull/2/commits/01c6d1fbc5047b25bb5d861a430ce41ccc834f59)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # NWERC 2022 Website
-This is the website for NWERC 2022
+This is the website for NWERC 2022, created with [Hugo](https://gohugo.io/).
+
+## Development
+- Clone this repository including submodules: `git clone --recurse-submodules <url>`
+- Run `hugo serve` to run the application
```
Comment (verwoerd) [inline README.md:6]: must be `hugo serve -D`, else it will serve the staticly build files

File: config.toml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 baseURL = 'https://2022.nwerc.eu/'
+copyright = 'NWERC'
```
Comment (verwoerd) [inline config.toml:2]: should be W.I.S.V. `Christiaan Huygens',  NWERC is not an entity.

File: content/contest.md [hunk 3]
```diff
@@ -7,24 +7,23 @@ menu: main
 type: page
 ---
 # The Contest
-The Northwestern Europe Regional Contest (NWERC) is a contest in which teams from universities all over the Northwestern part of Europe are served a series of algorithmic problems. The goal of each team is to solve as many problems as possible within the 5 hour time limit. Potential solutions are submitted and corrected by an automated judging system. The team that solves the most problems at the end of the contest qualify for the ICPC World Finals.
+The Northwestern Europe Regional Contest (NWERC) is a contest in which teams from universities all over the Northwestern part of Europe are served a series of algorithmic problems. The goal of each team is to solve as many problems as possible within the 5 hour time limit. Potential solutions are submitted and corrected by an automated judging system. The teams that solve the most problems at the end of the contest qualify for the ICPC World Finals.
```
Comment (verwoerd) [inline content/contest.md:10]: officially it is team(non-plural)

Comment (mpsijm) [reply content/contest.md]: Ah, then I fixed it the wrong way around :stuck_out_tongue: Thanks! :smile:

**Review by robertdijk — APPROVED** [review]
Love it

**Pull request merged**